src/__tests__/getUser.test.ts

```diff
@@ -0,0 +1,30 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as apiClient from '../';
+import { users as fixtures } from './fixtures/users';
+
+const mock = new MockAdapter(axios);
+
+afterEach(() => {
+	mock.reset();
+});
+
+const apiClientInit = new apiClient.APIClient({ apiBase: 'unics-social' });
+
+test('getUser(): fetches users correctly', async () => {
+	for (const fixture of fixtures) {
+		mock.onGet(`unics-social/users/${fixture.id}`).reply(200, {
+			status: 200,
+			error: '',
```

Comment:
You can safely remove status and error here. `status` in the response body is a feature of hs_auth, but we don't use that in the KB API. We also only respond with an error property if an error occurs.